server/src/main/java/com/linecorp/centraldogma/server/plugin/PluginInitContext.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.server.plugin;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.centraldogma.server.CentralDogmaConfig;
+
+/**
+ * A context that is used to pass when calling {@link AllReplicasPlugin#init(PluginInitContext)}.
+ */
+public final class PluginInitContext {
+
+    private final CentralDogmaConfig config;
+    private final ServerBuilder serverBuilder;
+
+    /**
+     * Creates a new instance.
+     */
+    public PluginInitContext(CentralDogmaConfig config, ServerBuilder serverBuilder) {
```

Comment:
```suggestion
    PluginInitContext(CentralDogmaConfig config, ServerBuilder serverBuilder) {
```

Reply:
Oops, I cannot remove the public modifier because the class is in a different package.